core/os/file/path_test.go

```diff
@@ -50,3 +53,32 @@ func TestPathContains(t *testing.T) {
 			Test(test.dir.Contains(test.file) == test.expected)
 	}
 }
+
+func TestIsExecutable(t *testing.T) {
+	ctx := log.Testing(t)
+	var (
+		tmpdir  string
+		err     error
+		tmpfile *os.File
+	)
+	tmpdir, err = ioutil.TempDir("/tmp", "path_test-")
```

Comment:
use `""` instead of `"/tmp"`, so that `ioutil.TempDir` uses the default temporary directory.